event_files/rvv_spec_metrics.json

```diff
@@ -0,0 +1,72 @@
+[
+    {
+        "MetricName": "RVV_RETIRED.FLOPC_SINGLE",
+        "BriefDescription": "Vector single-precision floating point operations executed per cycle",
+        "MetricExpr": "RVV_RETIRED.ELEMENT.FP_SINGLE / GEN.CYCLES"
+    },
```

Comment:
Looks like you forgot to switch the metric/event names from RVV_RETIRED to RVV_SPEC here